srcpkgs/paraview/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'paraview'
+pkgname=paraview
+version=5.6.1
+revision=1
+archs="x86_64"
```

Comment:
archs should only be used if there is a strict reason that it will never be supported on other archs (like musl specific packages or binary packages that are only available for certain architectures) or to specify `noarch`. For any other cases it is preferred to use `broken=` or `nocross=` with a reason why it is not working for these cases. `archs=="x86_64"` is in my experience nearly always wrong for packages that are build from source.

Reply:
Alright, removed `archs`. Thanks!